snc.sh

```diff
@@ -257,6 +257,7 @@ ${YQ} write --inplace ${INSTALL_DIR}/manifests/cluster-ingress-02-config.yml spe
 # This is only valid for openshift 4.3 onwards
 ${YQ} write --inplace ${INSTALL_DIR}/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value[domainMemory] 14336
 ${YQ} write --inplace ${INSTALL_DIR}/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value[domainVcpu] 6
+${YQ} write --inplace ${INSTALL_DIR}/openshift/99_openshift-cluster-api_master-machines-0.yaml spec.providerSpec.value.volume[volumeSize] 32212254720
```

Comment:
This is 30GiB (might be worth a comment, as  32212254720 is hard to parse ;), We used to have 31GiB I think.
You could also be more specific in the commit log about the openshift versions this is going to work with, so that people find answers to questions like "I'm generating 4.4 images, will this negatively impact me?".
We are extracting openshift-install from released bare metal container images, do these already have the required change?

Reply:
> We are extracting openshift-install from released bare metal container images, do these already have the required change?

from master side it already have those changes, on 4.5 side it will be part of next rc since we just miss one day to get it merge on 4.5 branch.

Reply:
I'd document this in the commit log so that we can have a clue why resizing no longer happens on 4.4

Reply:
I usually branch out snc for each openshift release so the user don't worry about what works and what not with master branch, I can add the note in commit.